website/docs/r/app_service_source_control_token.html.markdown

```diff
@@ -0,0 +1,37 @@
+---
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_app_service_source_control_token"
+sidebar_current: "docs-azurerm-resource-app-service-source-control-token"
+description: |-
+  Manages an App Service source control token.
+
+---
+
+# azurerm_app_service_source_control_token
+
+Manages an App Service source control token.
```

Comment:
It might be good to have a note here  how this is global to the subscription/type?

Reply:
Have made an attempt to fix this, but it might need to be adjusted to proper English, sorry.